### PR TITLE
[FLINK-13209] Following FLINK-12951: Remove TableEnvironment#sql and …

### PR DESCRIPTION
**Pull request by danny0405**
## What is the purpose of the change

This patch move create table ddl support from `#sql` to `#sqlUpdate`, also remove the deprecation of `sqlUpdate` and `sqlQuery`

### CODE REVIEW

File: flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java [hunk 1]
```diff
@@ -430,11 +362,51 @@ static TableEnvironment create(EnvironmentSettings settings) {
...
+	 * For example, the below DDL statement would create a CSV table named `tbl1`
+	 * into the current catalog:
+	 * <blockquote><pre>
+	 *    create table tbl1(
+	 *      a int,
+	 *      b bigint,
+	 *      c varchar
+	 *    ) with (
+	 *      connector = 'csv',
+	 *      csv.path = 'xxx'
```
Comment (wuchong) [inline flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java]: Give a validate with property? 
```
connector.type = 'filesystem',
connector.property-version = '1',
connector.path = '/path/to/file',
format.type = 'csv',
format.property-version = '1',
format.derive-schema = 'true'
```
However, I'm not sure the above properties work.

**Comment by KurtYoung** [comment]
cc @twalthr

File: flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java (hunk 1 above)
Comment (twalthr) [reply flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java]: I agree with Jark. If we show examples, then examples that also work in the current version.

File: flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java [hunk 2]
```diff
@@ -430,11 +362,51 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 * }
 	 * </pre>
 	 *
-	 * @deprecated Use {@link #sql(String)}.
+	 * <p>A DDL statement can also execute to create a table:
```
Comment (twalthr) [inline flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java]: Rephrase: "DDL statements can be executed to create tables:"

**Comment by KurtYoung** [comment]
+1 from my side, waiting for travis

**Pull request closed without merging**